designer/server/src/plugins/routes/api.test.js

```diff
@@ -67,12 +52,16 @@ describe('Server API', () => {
       }
     }
 
-    jest.spyOn(Wreck, 'get').mockResolvedValue({
-      payload: Buffer.from(JSON.stringify({}))
-    })
+    jest
+      .mocked(forms.getDraftFormDefinition)
+      .mockReturnValue(
+        Promise.resolve(
+          /** @type {import('@defra/forms-model').FormDefinition} */ ({})
+        )
+      )
```

Comment:
Mind using `.mockRejectedValueOnce()` for promises?

Would the response be an empty but valid `FormDefinition` with an HTTP 500?

Reply:
good spot

Reply:
Cheers. Don't need need the `@type` now either

I'll get this PR approved, but feel free to delete the test entirely if it's impossible to get an empty `{}` back